GOVERNANCE.md

```diff
@@ -122,3 +122,10 @@ If an agenda item cannot reach a consensus, a CTC member can call for either a
 closing vote or a vote to table the issue to the next meeting. The call for a
 vote must be approved by a simple majority of the CTC or else the discussion
 will continue.
+
+A vote may be scheduled for a future date to allow time for CTC members to
+better educate themselves about relevant issues. A vote is not valid unless
+there is participation of a quorum of the CTC. A CTC quorum is more than half of
+the CTC. Absentee votes may be cast in advance by commenting on the weekly CTC
```

Comment:
@addaleax Yes, that is correct. Or at least, that is my intention. I guess we'll see if other folks agree with that or not...

So:

The vote-to-schedule-a-vote in the previous paragraph requires more than half of the CTC. (I actually think that should be changed to half of the attendees and that there needs to be a quorum, but I don't feel strongly enough about it to try to bundle it into this PR.)

Once there's an actual vote on a decision on a contentious issue, *that* vote is decided by simple majority (or even tiny plurality!) of the participants as long as there is a quorum participating.

So, for example, there are currently 18 people on the CTC. Let's say there was a contentious issue around whether we should deprecate a particular API or not. And a vote is scheduled to decide. Let's say 8 people don't show up to the meeting and do not register a vote in the GitHub issue for the meeting agenda. Let's say the remaining 10 people all vote either in the issue tracker or else show up at the meeting to register a vote. Hooray, that's a qurorum, the vote counts! Let's say 7 of those 10 people say, "I looked at the issue and I just don't have a strong opinion either way. I abstain. I want a decision to be made, but I don't care what that decision is." That leaves only 3 people casting non-abstention votes. Let's say 2 of them vote for deprecation and 1 votes against. Then it's time to deprecate that API.

Of course, that scenario would be hopefully exceedingly rare. We don't actually have too many votes in my experience (and when we do, they almost always end up being very lopsided). But the point of the quorum rule isn't to make certain that a large majority get their way. It's too simply guarantee *sufficient* support that the decision has legitimacy without the actual decision-making process being a source of dramatic delays or inaction. It's an absolute floor for what a decision might look like in the absolute worst of times.

In reality, I think we tend to work pretty hard to get at least 10 votes for anything, so the scenario I describe above is highly unlikely to ever actually occur.